datajoint/blob.py

```diff
@@ -141,6 +141,7 @@ def read_blob(self, n_bytes=None):
                 "F": self.read_recarray,  # numpy array with fields, including recarrays
                 "d": self.read_decimal,  # a decimal
                 "t": self.read_datetime,  # date, time, or datetime
+                "T": self.read_np_datetime,  # np.datetime64
```

Comment:
ok, this is a tough one for us to accept. We are designing a general, language-agnostic data serialization protocol. A reference to numpy here is not appropriate.

You could use `int64_datetime` instead. Please look into whether this convention to serialize datetimes is used in other languages.